democracy/models/label.py

```diff
@@ -1,10 +1,10 @@
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 
-from .base import BaseModel
+from .base import StringIdBaseModel
 
 
-class Label(BaseModel):
+class Label(StringIdBaseModel):
```

Comment:
And all in all, I'm not quite sure what this change is supposed to achieve. The commit message is quite opaque (and lacks any sort of issue ref).